examples/python/initialize.py

```diff
@@ -55,7 +55,7 @@
 
 # Desired vector
 print("Desired probabilities: ")
-print(list(map(lambda x: format(abs(x * x), ".3f"), desired_vector)))
+print([format(abs(x * x), ".3f") for x in desired_vector])
```

Comment:
The Python looks fine here, but the maths looks questionable!  I guess it happens to work since the statevector entries are either purely real or purely imaginary.